cp-overrides.yaml

```diff
@@ -5,7 +5,12 @@ modules:
           - name: cct_module
             git:
                   url: https://github.com/jboss-openshift/cct_module.git
-                  ref: sprint-18
+                  ref: sprint-22
+          - name: jboss-eap-modules
+            git:
+                  url: https://github.com/jboss-container-images/jboss-eap-modules
+                  ref: sprint-22
```

Comment:
I don't see any sprint-22 tag or branch in this repo.

Reply:
Yep, I need to make those after the sprint builds.

I thought these would be ok in -dev until the next time we release, meh, now I'm thinking about it, we're probably just better off waiting for the actual release to update these in the -cp and -rel branches.

Reply:
What I mean is, we should add as part of the release merge, to update the -cp and -rel overrides vs trying to track them with sprint builds.